### PR TITLE
BMP v2 swd flashing issue fix for lisa mxs

### PR DESCRIPTION
**Pull request by kevindehecker**
Made lisa mxs compatible with BMP v2 by disabling voltage sensing. 
This required making a seperate board file for the mxs. Hence also changed the ladylisamxs airframe file to use the new board. Also added a cx10 airframe file.

### CODE REVIEW

**Comment by podhrmic** [comment]
@kevindehecker what is Lisa MXS? Did you mean Lisa S? https://wiki.paparazziuav.org/wiki/Lisa/S

File: conf/maps_example.xml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <!DOCTYPE maps SYSTEM "maps.dtd">
 
-<maps google_version="733"/>
+<maps google_version="129"/>
```
Comment (gautierhattenberger) [inline conf/maps_example.xml]: please don't change this file (you should rebase on master)

File: conf/boards/lisa_mxs_1.0.makefile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# lisa_mxs_1.0.makefile
+#
+# https://wiki.paparazziuav.org/wiki/Lisa/MXS_v1.0
+#
+
+BOARD=lisa_mxs
+BOARD_VERSION=1.0
+BOARD_CFG=\"boards/$(BOARD)_$(BOARD_VERSION).h\"
+
+ARCH=stm32
```
Comment (gautierhattenberger) [inline conf/boards/lisa_mxs_1.0.makefile]: You're adding a lot of file related to ChibiOS, but still selecting the stm32 arch. And I don't think it is currently working without ChibiOS at the moment as some definition are missing. I think you should limit your pull request to adding the basic support of the lisa_mxs board and its flashing option, and add ChibiOS support later it needed when it is ready.

**Comment by dewagter** [comment]
This flashing options should also be the default for the lisa-s. It has the
same problem with BMPv2.

On Aug 20, 2017 22:50, "Gautier Hattenberger" <notifications@github.com>
wrote:

*@gautierhattenberger* requested changes on this pull request.
------------------------------

In conf/maps_example.xml
<https://github.com/paparazzi/paparazzi/pull/2104#discussion_r134125893>:

> @@ -1,3 +1,3 @@
 <!DOCTYPE maps SYSTEM "maps.dtd">

-<maps google_version="733"/>
+<maps google_version="129"/>

please don't change this file (you should rebase on master)
------------------------------

In conf/boards/lisa_mxs_1.0.makefile
<https://github.com/paparazzi/paparazzi/pull/2104#discussion_r134125983>:

> @@ -0,0 +1,28 @@
+# Hey Emacs, this is a -*- makefile -*-
+#
+# lisa_mxs_1.0.makefile
+#
+# https://wiki.paparazziuav.org/wiki/Lisa/MXS_v1.0
+#
+
+BOARD=lisa_mxs
+BOARD_VERSION=1.0
+BOARD_CFG=\"boards/$(BOARD)_$(BOARD_VERSION).h\"
+
+ARCH=stm32

You're adding a lot of file related to ChibiOS, but still selecting the
stm32 arch. And I don't think it is currently working without ChibiOS at
the moment as some definition are missing. I think you should limit your
pull request to adding the basic support of the lisa_mxs board and its
flashing option, and add ChibiOS support later it needed when it is ready.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/paparazzi/paparazzi/pull/2104#pullrequestreview-57382041>,
or mute the thread

File: conf/boards/lisa_mxs_1.0.makefile (hunk 2 above)
Comment (kevindehecker) [reply conf/boards/lisa_mxs_1.0.makefile]: Removed those files. I just copied it assuming it already worked.

**Comment by kevindehecker** [comment]
@dewagter  The lisa_s now also uses the new bmp setting.
@podhrmic  I'm planning to add some stuff on the wiki eventually ,but first figure it all out myself :)

**Pull request merged**